jsx/package.json

```diff
@@ -7,8 +7,8 @@
   "license": "BSD-3-Clause",
   "scripts": {
     "build": "webpack",
+    "build:watch": "webpack watch",
     "hot": "webpack && webpack-dev-server",
-    "place": "cp build/admin-react.js* ../share/jupyterhub/static/js/",
```

Comment:
this helped development, because I could run `npm run build:watch` with a real, running jupyterhub and keep reloading. I found this more useful than `npm run hot`, which doesn't use real JupyterHub responses.